CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to qrm
+
+## Before You Start
+
+- Make sure there's an issue for the feature, bugfix, or other improvement you want to make.
+- Make sure it's something that the project maintainers want.
+  We can discuss it and assign the issue to you.
+- Make sure work isn't already being done on the issue.
+
+### Environment Setup
+
+Once all of the above is done, you can get started by setting up your development envronment.
+
+1. [Fork this repo][1] into your own GitHub namespace.
+1. Make sure the `master` branch is up to date, then make yourself a new branch with a descriptive name.
+1. Once the forked repo is cloned and on the proper branch, you can set up the development environment.
+    1. Run `make dev-install`.
+       This should install everything you need to develop and run qrm.
```

Comment:
Not true, python3.7+ is required, and 3.8+ if we decide to bump the version requirement up.

Reply:
ah yes true